test/utilities/sideBar.ts

```diff
@@ -118,25 +118,26 @@ export async function getVisibleItems(
   return [...rows.values()];
 }
 
-export async function retrieveAllApexTestItemsFromSidebar(
+export async function retrieveAllTestItemsFromSidebar(
```

Comment:
Good refactor - thanks

Reply:
One nit, function name `retrieveAllTestItemsFromSidebar` implies that all tests in the section are being fetched, while the implementation will stop the fetch when the number of tests found === expectedNumTests.

Is there a better name for the function?